.github/actions/contentful-checks/action.yml

```diff
@@ -0,0 +1,148 @@
+name: Contentful Checks
+description: Defines outputs to set up Contentful-based steps
+
+inputs:
+  pr-number:
+    description: 'PR Number'
+    required: true
+  contentful-space-id:
+    description: 'Contentful Space ID'
+    required: true
+  contentful-master-env:
+    description: 'Contentful master environment'
+    required: true
+  contentful-token:
+    description: 'Contentful access token'
+    required: true
+  max-contentful-envs:
+    description: 'Max Contentful environments'
+    required: true
+
+outputs:
+  contentful-env:
+    description: 'Contentful environment to be used on branch'
+    value: ${{ steps.determine-environment.outputs.env-id }}
+  on-branch-env:
+    description: 'Whether the Contentful environment being used is a branch environment'
+    value: ${{ steps.determine-environment.outputs.on-branch-env }}
+  contentful-limit-reached:
+    description: 'Whether Contentful environment limit has been reached'
+    value: ${{ steps.check-env-limit.outputs.limit-reached }}
+  contentful-env-exists:
+    description: 'Whether the Contentful environment already exists'
+    value: ${{ steps.check-contentful-exists.outputs.exists }}
+  contentful-env-migrations-up-to-date:
+    description: 'Whether the Contentful environment is up to date with local migrations'
+    value: ${{ steps.check-contentful-migrated.outputs.migrations-up-to-date }}
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Determine whether migrations need to be run on master environment
+      id: count-outstanding-migrations
+      shell: bash
+      run: |
+        OUTSTANDING_MIGRATIONS=$(yarn workspace @asap-hub/contentful space:migrate:crn:count)
+        echo "OUTSTANDING_MIGRATIONS=$OUTSTANDING_MIGRATIONS" >> $GITHUB_ENV
+      env:
+        CONTENTFUL_SPACE_ID: ${{ inputs.contentful-space-id }}
+        CONTENTFUL_ENV_ID: ${{ inputs.contentful-master-env }}
+        CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: ${{ inputs.contentful-token }}
+    - name: Determine Environment
+      id: determine-environment
+      shell: bash
+      run: |
+        CONTENTFUL_ENV=$CONTENTFUL_MASTER_ENV
+        BRANCH_ENV=False
+
+        if [ $OUTSTANDING_MIGRATIONS != "0" ]; then
+          CONTENTFUL_ENV=crn-$PR_NUMBER
+          BRANCH_ENV=True
+        fi
+
+        echo "$OUTSTANDING_MIGRATIONS migrations to run on $CONTENTFUL_ENV."
+        echo "::set-output name=env-id::$CONTENTFUL_ENV"
+        echo "::set-output name=on-branch-env::$BRANCH_ENV"
```

Comment:
BRANCH_ENV is a boolean. When we export it, on line 65. It will be a string?

Reply:
Env variables are always strings on POSIX systems to the best of my knowledge, we can just do a string check.